src/chat/zulip.md

```diff
@@ -0,0 +1,180 @@
+# Zulip
+
+[Rust's Zulip](rust-lang.zulipchat.com) is used by a number of teams, notably the compiler and language
+teams, along with their working groups.
```

Comment:
You can now add Libs here :) https://internals.rust-lang.org/t/the-library-team-is-moving-from-irc-to-zulip/11598